src/scripts/popup/index.ts

```diff
@@ -10,8 +10,23 @@ import {browser} from 'webextension-polyfill-ts';
 const port: browser.runtime.Port = browser.runtime.connect();
 
 
-utils.queryOrThrow('#keybinds').addEventListener('click', (): void => {
-	port.postMessage(tabbo.PopUpCommand.KEYBINDS);
+utils.queryOrThrow('#keybinds').addEventListener('click', async (): void => {
+	/* FIXME
+	 * Chrome doesn't support `browser.runtime.getBrowserInfo` so we can't use that!
+	 * When it does, change to that.
+	 */
+	const browserType: utils.Browser = utils.checkBrowser();
+	alert(browserType);
```

Comment:
can we please remove this before merging?

Reply:
oops